bugsnag-plugin-android-ndk/src/main/assets/include/bugsnag.h

```diff
@@ -36,6 +39,14 @@ void bugsnag_set_user_env(JNIEnv *env, char* id, char* email, char* name);
 void bugsnag_leave_breadcrumb(char *message, bsg_breadcrumb_t type);
 void bugsnag_leave_breadcrumb_env(JNIEnv *env, char *message, bsg_breadcrumb_t type);
 
+void bugsnag_add_on_error(bsg_on_error on_error);
+void bugsnag_add_on_error_env(JNIEnv *env, bsg_on_error on_error);
+void bugsnag_remove_on_error(bsg_on_error on_error);
+void bugsnag_remove_on_error_env(JNIEnv *env, bsg_on_error on_error);
```

Comment:
As discussed IRL, there doesn't seem any point in adding the `_env` variant unless the method actually requires access to the JNI.